aspnetcore/blazor/javascript-interoperability/call-javascript-from-dotnet.md

```diff
@@ -657,34 +657,79 @@ Other data types, such as string arrays, can be converted but require creating a
 
 ## Stream from JavaScript to .NET
 
-Blazor supports streaming data directly from JavaScript to .NET. Streams are requested using the `IJSStreamReference` interface:
+Blazor supports streaming data directly from JavaScript to .NET. Streams are requested using the `Microsoft.JSInterop.IJSStreamReference` interface.
 
-In client-side JavaScript:
+`Microsoft.JSInterop.IJSStreamReference` returns a <xref:System.IO.Stream> and uses the following parameters:
```

Comment:
Not sure of doc convention here, but maybe:

```suggestion
`Microsoft.JSInterop.IJSStreamReference.OpenReadStreamAsync` returns a <xref:System.IO.Stream> and uses the following parameters:
```

https://github.com/dotnet/aspnetcore/blob/66ceea4fc6ccbd710c5b77782761f92557a08be6/src/JSInterop/Microsoft.JSInterop/src/IJSStreamReference.cs#L27